xds/src/test/java/io/grpc/xds/XdsSecurityClientServerTest.java

```diff
@@ -146,6 +159,37 @@ public void tlsClientServer_noClientAuthentication() throws Exception {
     assertThat(unaryRpc(/* requestMessage= */ "buddy", blockingStub)).isEqualTo("Hello buddy");
   }
 
+  /** Use system root ca cert for TLS channel - no mTLS. */
+  @Test
+  public void tlsClientServer_useSystemRootCerts() throws Exception {
+    System.setProperty( "javax.net.ssl.trustStore", getCacertFilePathForTestCa());
+    System.setProperty( "javax.net.ssl.trustStorePassword", "changeit");
+    System.setProperty("javax.net.ssl.trustStoreType", "JKS");
+    DownstreamTlsContext downstreamTlsContext =
+        setBootstrapInfoAndBuildDownstreamTlsContext(null, null, null, null, false, false);
+    buildServerWithTlsContext(downstreamTlsContext);
+
+    // for TLS, client only needs trustCa
```

Comment:
This comment looks to be because `false` was passed for hasIdentityCert. You copied it, but it doesn't seem applicable with how this is being called.

Reply:
Done.